build/common.props

```diff
@@ -1,10 +1,11 @@
 <Project>
   <PropertyGroup>
     <!-- Packages can have independent versions and only increment when released -->
-    <Version>3.0.29$(VersionSuffix)</Version>
+    <Version>3.1.0$(VersionSuffix)</Version>
     <ExtensionsStorageVersion>4.0.5$(VersionSuffix)</ExtensionsStorageVersion>
-    <HostStorageVersion>4.0.2$(VersionSuffix)</HostStorageVersion>
+    <HostStorageVersion>4.1.0$(VersionSuffix)</HostStorageVersion>
     <LoggingVersion>4.0.2$(VersionSuffix)</LoggingVersion>
+    <StorageProviderVersion>1.0.0$(VersionSuffix)</StorageProviderVersion>
```

Comment:
These need to be flagged as beta if we're expecting them to depend on beta packages.